store/loomethdb.go

```diff
@@ -111,13 +116,15 @@ func (b *batch) Write() error {
 		return bytes.Compare(b.cache[j].key, b.cache[k].key) < 0
```

Comment:
This cache is no longer needed, should just write to the batch directly. Only the `LogBatch` needs the cache & sorting to ensure log entries are written out in deterministic order.